jaeger-core/src/main/java/com/uber/jaeger/samplers/RemoteControlledSampler.java

```diff
@@ -22,29 +22,31 @@
 package com.uber.jaeger.samplers;
 
 import com.uber.jaeger.exceptions.SamplingStrategyErrorException;
-import com.uber.jaeger.exceptions.UnknownSamplingStrategyException;
 import com.uber.jaeger.metrics.Metrics;
+import com.uber.jaeger.samplers.http.OperationSamplingParameters;
 import com.uber.jaeger.samplers.http.ProbabilisticSamplingStrategy;
 import com.uber.jaeger.samplers.http.RateLimitingSamplingStrategy;
 import com.uber.jaeger.samplers.http.SamplingStrategyResponse;
-import com.uber.jaeger.samplers.http.SamplingStrategyType;
 
 import java.util.Timer;
 import java.util.TimerTask;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
 
 @SuppressWarnings("EqualsHashCode")
 @ToString(exclude = {"pollTimer", "lock"})
```

Comment:
This is preexisting code, not part of this commit. 
It is used for initialization logging to what the sampler has been initialized with.